spec/ares/ares_spec.rb

```diff
@@ -6,7 +6,7 @@
   end
 
   let(:valid_ico) { '27169278' }
-  let(:nonexistent_ico) { '00000011' }
+  let(:nonexistent_ico) { '00000001' }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.